reddit-background

```diff
@@ -533,6 +533,15 @@ def _get_northern_hemisphere_season():
     else:
         return 'winter'
 
+def slugify(value):
+    """
+    Normalizes string, converts to lowercase, removes non-alpha characters,
+    and converts spaces to hyphens.
+    """
```

Comment:
Would be a good idea to place an attribution line in the docstring, something like

    source: https://stackoverflow.com/questions/295135/turn-a-string-into-a-valid-filename/295466#295466